src/main/java/frc/robot/commands/PlaceCone/Rotate.java

```diff
@@ -165,7 +167,7 @@ public double[] chassisValuesLower() {
     x[2] =
         angleDone
             ? 0
-            : -((poseEstimatorSubsystem.getAngle() % 180)
+           : -((poseEstimatorSubsystem.getAngle() % 180)
```

Comment:
This math is a bit weird too.  You need to think thru the equations with example values of `poseEstimatorSubsystem.getAngle()` and compute what will come out.  Then will be able to verify you got it right or wrong